src/utils/thirdparty_env.js

```diff
@@ -0,0 +1,26 @@
+import getResourcesPath from './resources'
+
+var thirdpartyPath = getResourcesPath('thirdparty'),
+  env = {};
+
+if (process.platform == 'darwin') {
```

Comment:
`(process.platform == 'darwin')` this should be `(process.platform === 'darwin')`